RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts.swift

```diff
@@ -173,19 +173,8 @@ public struct ImportMnemonicControllingAccounts: Sendable, FeatureReducer {
 			return .none
 
 		case .confirmSkippingBDFS(.delegate(.confirmed)):
-			loggerGlobal.notice("Skipping BDFS! Generating a new one and hiding affected accounts/personas.")
 			state.destination = nil
-			return .run { [entitiesControlledByFactorSource = state.entitiesControlledByFactorSource] send in
-				loggerGlobal.info("Generating mnemonic for new main BDFS")
-				let newMainBDFS = try await factorSourcesClient.createNewMainBDFS()
```

Comment:
BDFS will be created by Sargon when the profile is imported and the skippedBDFS flag is true